agent/unit-test/softwarecontaineragent_unittest.cpp

```diff
@@ -32,7 +32,7 @@ class SoftwareContainerAgentTest: public ::testing::Test
     SoftwareContainerAgentTest() { }
     std::shared_ptr<SoftwareContainerAgent> sca;
     Glib::RefPtr<Glib::MainContext> m_context = Glib::MainContext::get_default();
-    int m_preloadCount = 1;
+    int m_preloadCount = 0;
```

Comment:
Sure, could you add a small sanity check test trying to start a preloaded container?